SoObjects/SOGo/SOGoUserManager.m

```diff
@@ -768,14 +776,17 @@ - (void) _fillContactInfosForUser: (NSMutableDictionary *) currentUser
             c_imaplogin = [userEntry objectForKey: @"c_imaplogin"];
           if (!c_sievehostname)
             c_sievehostname = [userEntry objectForKey: @"c_sievehostname"];
-          access = [[userEntry objectForKey: @"CalendarAccess"] boolValue];
-          if (!access)
-            [currentUser setObject: [NSNumber numberWithBool: NO]
-                            forKey: @"CalendarAccess"];
-          access = [[userEntry objectForKey: @"MailAccess"] boolValue];
-          if (!access)
-            [currentUser setObject: [NSNumber numberWithBool: NO]
-                            forKey: @"MailAccess"];
+
+          enumerator = [access_types_list objectEnumerator];
+          while ((access_type = [enumerator nextObject]) != nil)
+            {
+              access = [[userEntry objectForKey: access_type] boolValue];
+              if (!access)
+                {
```

Comment:
The current code style is to not put brackets here (simple if's with only one line)